### PR TITLE
make resource name meaning and validation more obvious

### PR DESCRIPTION
**Pull request by grosser**
before: 

no explanation and no idea why it was wrong
screen shot 2016-09-05 at 6 16 23 pm

after:
explanation + matching in the UI with pattern help

screen shot 2016-09-05 at 6 08 27 pm
screen shot 2016-09-05 at 6 15 49 pm

@zendesk/samson

### CODE REVIEW

**Comment by grosser** [comment]
@livathinos

**Comment by grosser** [comment]
also prefilling it:

screen shot 2016-09-06 at 1 30 58 pm

File: plugins/kubernetes/app/views/kubernetes/roles/_form.html.erb [hunk 1]
```diff
@@ -24,9 +25,14 @@
     </div>
 
     <div class="form-group">
-      <%= form.label :resource_name, class: "col-lg-2 control-label" %>
+      <% pattern = Kubernetes::RoleVerifier::VALID_LABEL.source.sub!('\\A', '^').sub!('\\z', '$') || raise %>
```
Comment (livathinos) [inline plugins/kubernetes/app/views/kubernetes/roles/_form.html.erb]: Maybe move this logic inside the `RoleVerifier` class?

**Comment by livathinos** [comment]
Besides a comment, :+1:

**Pull request merged**